core/src/main/scala-2.12/cats/data/OneAndLowPriority4.scala

```diff
@@ -5,6 +5,7 @@ import scala.annotation.tailrec
 import scala.collection.mutable.Builder
 
 abstract private[data] class OneAndLowPriority4 {
+  @deprecated("Will remove OneAnd in cats v3", "2.6.1")
```

Comment:
I suggest to change the deprecation message to something like:

> For collections, use special NonEmpty types instead of OneAnd (see #3089 for details).

Reply:
This has been added to the PR.